public/js/event-views.js

```diff
@@ -659,7 +1082,7 @@ views.ChatInputView = Backbone.Marionette.ItemView.extend({
 
     chat: function(e) {
         var msg = this.ui.chatInput.val();
-        var postAsAdmin = IS_ADMIN && this.ui.asAdmin.is(":checked");
+        var postAsAdmin = e && this.ui.asAdmin.is(":checked");
```

Comment:
What's going on here?  ``IS_ADMIN`` check is probably unnecessary since it duplicates the ``asAdmin`` check, as ``asAdmin`` will only be shown if ``IS_ADMIN`` is true.  But not sure why we need the ``e &&`` bit.